seatunnel-ci-tools/src/test/java/org/apache/seatunnel/api/ConnectorOptionCheckTest.java

```diff
@@ -233,6 +233,8 @@ private Set<String> buildWhiteList() {
         whiteList.add("OracleIncrementalSourceOptions");
         whiteList.add("MySqlIncrementalSourceOptions");
         whiteList.add("MongodbIncrementalSourceOptions");
+        whiteList.add("LocalFileSourceOptions");
+        whiteList.add("LocalFileSinkOptions");
```

Comment:
I can the UT is missing `BaseMultipleTableFileSink` extendedType check. Can you help add it and update related code, thanks.